core/src/main/scala/cats/UnorderedFoldable.scala

```diff
@@ -14,6 +15,9 @@ import simulacrum.{noop, typeclass}
   def unorderedFold[A: CommutativeMonoid](fa: F[A]): A =
     unorderedFoldMap(fa)(identity)
 
+  def unorderedReduceOption[A](fa: F[A])(implicit A: CommutativeSemigroup[A]): Option[A] =
+    unorderedFoldMap(fa)(_.some)(cats.instances.option.catsKernelStdCommutativeMonoidForOption(A))
```

Comment:
Do you mind changing this to `Some(_)` (adding a type parameter to `unorderedFoldMap` if needed)? We're trying to avoid using `syntax` in core, to keep things simple and minimize cyclic dependencies between packages.

Reply:
Type inference fails with `Some(_)` which is why I resorted to `_.some`. Should I use `Some(_): Option[A]`?

Reply:
> I'll take a closer look at the bincompat issues ASAP (probably this weekend). I think you'll probably also hit bincompat breakage with these laws changes, but that's probably less annoying to work around.

Hi @travisbrown. Would it make sense to revisit this for Cats 2.2?